interop/authzen-interop-website/docs/spec-1.0/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
instead of creating two different versions of the docs, could we instead create two different scenarios (e.g. under /docs/scenarios/todo.md and /docs/scenarios/todo-boxcarred.md), and do the same thing for the results pages?